packages/botfuel-dialog/package.json

```diff
@@ -18,7 +18,9 @@
     "botfuel-clean": "./build/clean.js"
   },
   "scripts": {
-    "compile": "rm -rf build; ../../node_modules/.bin/babel src --out-dir build; cp -r src/corpora/*.txt build/corpora/",
```

Comment:
Why do we no longer need compile?

Reply:
it changed to "build" now. Normally you do not call it directly but it is called in yarn bootstrap, yarn test